zerver/tests/test_messages.py

```diff
@@ -203,13 +204,13 @@ def assert_invalid_email() -> Any:
         user3_email = 'user3@3.example.com'
         feedback_email = 'feedback@zulip.com'
         support_email = 'support@3.example.com'  # note: not zulip.com
-
+        welcome_bot_email = "welcome-bot@zulip.com"
         user1 = self.create_user(user1_email)
         user1a = self.create_user(user1a_email)
         user2 = self.create_user(user2_email)
         self.create_user(user3_email)
         feedback_bot = get_system_bot(feedback_email)
-        support_bot = self.create_user(support_email)
```

Comment:
I think with this change `support_email` no longer serves its intended role in this test.

In particular the "old loophole" case below wants to refer to the email of an actually existing cross-realm bot user, so I think that test case no longer functions. Maybe we no longer need that regression test -- that issue seems less likely with the way our design has evolved -- but in that case we should just delete it.